resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/KubernetesClusterManager.scala

```diff
@@ -65,6 +68,7 @@ private[spark] class KubernetesClusterManager extends ExternalClusterManager wit
       apiServerUri,
       Some(sc.conf.get(KUBERNETES_NAMESPACE)),
       authConfPrefix,
+      clientType,
```

Comment:
```scala
-      clientType,
+      SparkKubernetesClientFactory.ClientType.Driver,
```

Reply:
This is more consistent with `KubernetesClientApplication.scala`. In this file, `clientType` is always `Driver`.